coops-ui/src/components/dialogs/WarningPopover.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <v-menu v-if="dialogObj != null"
+    :close-on-content-click="false"
+    :nudge-width="200"
+    offset-y
+    max-width="18rem"
+  >
+    <template v-slot:activator="{ on }">
+      <span class="warning-text" v-on="on">
+        <v-icon class="warning-icon">mdi-information-outline</v-icon>{{ dialogObj.title }}
+      </span>
+    </template>
+    <v-card>
+      <v-list-item>
+        <v-list-item-content>
+         {{ dialogObj.msg }}
+        </v-list-item-content>
+      </v-list-item>
+    </v-card>
+  </v-menu>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+import { AlertMessage } from '@/interfaces'
+
+@Component({})
+export default class WarningPopover extends Vue {
+  @Prop({ default: null })
+  private dialogObj: AlertMessage
+}
+</script>
+
+<style lang="scss">
+@import '@/assets/styles/theme.scss';
+.warning-text {
+  font-weight: bold;
+  font-size: 14px;
+  color: $BCgovIconBlue;
+    .v-icon {
```

Comment:
This line should be at the same indent level as the line above it. Also, we try to add a blank line between style blocks, to make it easier to see that a new selector is declared.